librz/egg/egg_Cfile.c

```diff
@@ -150,9 +149,16 @@ static struct cEnv_t *rz_egg_Cfile_set_cEnv(RZ_BORROW RZ_NONNULL RzPath *sys_pat
 		use_clang = true;
 		cEnv->TEXT = "0.__TEXT.__text";
 	}
+	if (!strcmp(cEnv->TRIPLET, "windows-x86-32")) {
+		free(cEnv->TRIPLET);
+		cEnv->TRIPLET = rz_str_dup("windows-x86");
+	}
 
-	buffer = rz_str_newf("%s -fno-stack-protector -nostdinc -include '%s'/'%s'/sflib.h",
-		cEnv->CFLAGS, cEnv->SFLIBPATH, cEnv->TRIPLET);
+	buffer = rz_str_newf(
+		"%s -fno-stack-protector -nostdinc -include '%s/%s/sflib.h'",
```

Comment:
```suggestion
		"%s -fno-stack-protector -nostdinc -include '" RZ_JOIN_3_PATHS("%s", "%s, "sflib.h") "'",
```

In the original issue, the string of text generated was like: `C:\Users\USER\Desktop\repos\rz\include\librz/sflib'/'windows-x86-32'/sflib.h`

This will fix also the path output on windows to be like: `C:\Users\abinr\Desktop\repos\rz\include\librz\sflib\windows-x86-32\sflib.h`